javascript/javascript1/week2/event.js

```diff
@@ -0,0 +1,43 @@
+// console.log("inside event.");
+const now = new Date();
+let day = now.getDay();
+let newday;
+let calcweekday;
+
+function getDay(eventDay) {
+  const getremaindarWeekDay = eventDay % 7;
+  let weekday = day + getremaindarWeekDay;
+
+  if (weekday >= 7) {
+    calcweekday = weekday % 7;
+  } else {
+    calcweekday = weekday;
+  }
+
+  switch (calcweekday) {
+    case 0:
+      newday = "Sunday";
+      break;
+    case 1:
+      newday = "Monday";
+      break;
+    case 2:
+      newday = "Tuesday";
+      break;
+    case 3:
+      newday = "Wednesday";
+      break;
+    case 4:
+      newday = "Thursday";
+      break;
+    case 5:
+      newday = "Friday";
+      break;
+    case 6:
+      newday = "Saturday";
+      break;
+  }
```

Comment:
This is a very verbose way to do an array index lookup. Put all the weekdays into an array and then use `calcweekday` as its index.

You chould try to make this function independent of the global scope. It's possible

Reply:
Thanks for the suggestion. I made the changes accordingly and it is much easier way to write the function.